webaudio/the-audio-api/the-audiocontext-interface/processing-after-resume.https.html

```diff
@@ -26,30 +26,51 @@
     'the-audioworklet-interface/processors/port-processor.js';
 
 promise_test(async () => {
+  let construct1;
+  async function run_test() {
+    const timeBeforeResume = realtime.currentTime;
+    // Get the currentTime at the same time with AudioContext.currentTime.
+    const {node: node1} = await construct1;
+    const constructReply1 = await ping_for_reply(node1);
+
+    // Two AudioWorkletNodes are constructed.
+    // node1 is constructed before and node2 after the resume() call.
+    await realtime.resume();
```

Comment:
Awaiting before constructing provides less time between construction and the suspend, and so increases the likelihood that the no frames are processed.

The intention was that some frames would be processed so that the processed frame count and the advance in `currentTime` could be compared.